### DISCUSSION TITLE
Team name when submit?

### DISCUSSION THREAD

**Discussion by cuteyyt** [FETS-AI/Challenge#65] [Q&A] [answered]
Dear Organizers:
Thank you very much for your hard work in organizing this challenge.

Since the team name is required when we submit for task 2, but we forgot the team name when we registered, may I ask what we should do?

**Comment by sarthakpati** [comment, marked as answer]
Hi, 

Please email us at challenge@fets.ai with any details you can provide (full name, email you used to register, university/organization name) and we will follow up. 

Cheers,
Sarthak

**Reply by cuteyyt** [reply]
Ok, thanks!
